src/App.vue

```diff
@@ -90,7 +90,12 @@
       <!-- Don't show page if an error dialog is displayed. -->
       <main v-if="!isErrorDialog">
         <BreadCrumb :breadcrumbs="breadcrumbs" />
-        <EntityInfo class="py-5 px-3" />
+
+        <div id="entity-info-wrapper">
+          <v-container class="py-5">
+            <EntityInfo />
+          </v-container>
+        </div>
```

Comment:
Container needed to be wrapped in a div so that div can have white background while container auto-sizes according to screen width.